src/components/Slider/Slider.jsx

```diff
@@ -0,0 +1,242 @@
+import React, { forwardRef, useRef } from "react";
+import PropTypes from "prop-types";
+import { COLORS_BASIC, SIZES_BASIC } from "../../constants";
+import useMergeRefs from "../../hooks/useMergeRefs";
+import "./Slider.scss";
+import { bem } from "./SliderCommons";
+import { ensureValueDefault } from "./SliderHelpers";
+import { SliderProvider } from "./SliderContext";
+import SliderBase from "./SliderBase/SliderBase";
+import SliderInfix from "./SliderInfix";
+import { createBemBlockHelper } from "../../helpers/bem-helper";
+
+const Slider = forwardRef(
+  (
+    {
+      ariaLabel,
+      ariaLabeledBy,
```

Comment:
please change to ariaLabelledby (to be as close as possible to native html syntax aria-labelledby

Reply:
Fixed according to comment. 
(https://github.com/mondaycom/monday-ui-react-core/pull/424/commits/1031367619056ae9a182bb88a707aaeefd3c3f04)